router/router.go

```diff
@@ -127,10 +128,7 @@ func (rt *HandleT) workerProcess(worker *workerT) {
 		batchTimeStat.Start()
 		logger.Debugf("Router :: trying to send payload to %s. Payload: ", rt.destID, job.EventPayload)
 
-		userID := integrations.GetUserIDFromTransformerResponse(job.EventPayload)
-		if userID == "" {
```

Comment:
Do we need to remove this assertion? We've never hit this before, so can still keep it.